internal/driver/cli.go

```diff
@@ -280,6 +290,9 @@ var usageMsgSrc = "\n\n" +
 	"  Source options:\n" +
 	"    -seconds              Duration for time-based profile collection\n" +
 	"    -timeout              Timeout in seconds for profile collection\n" +
+	"    -key                  Specify TLS private key\n" +
```

Comment:
Ditto (I like the brevity of the docs but we should here also clarify that this about the TLS params for profile and symbol fetch).